gui/wxpython/startup/utils.py

```diff
@@ -87,3 +87,34 @@ def create_database_directory():
         pass
 
     return None
+
+
+def startup_location_exists():
+    """Check for demolocation directory in distribution.
+
+    Returns True if found or False if nothing was found.
+    """
+    home = os.path.expanduser("~")
+    demolocation = os.path.join(home, "grass", "demolocation"),
+    print(demolocation)
+
+    # Find out if demolocation exists
+    if os.path.exists(demolocation):
+        return True
+    return False
+
+
+def copy_startup_location(grassdatabase):
+    """Copy the simple demolocation with some data to GRASS database.
+    Returns True if successfully copied or False when an error was encountered.
+    """
+    home = os.path.expanduser("~")
+    demolocation = os.path.join(home, "grass", "demolocation")
+
+    # Copy source demolocation into GRASS database
+    try:
+        copytree(demolocation, grassdatabase, ignore=ignore_patterns('*.tmpl','Makefile*'))
```

Comment:
```suggestion
    home = os.path.expanduser("~")
    src = os.path.join(home, "grass", "demolocation")
    dst = os.path.join(grassdatabase, "demolocation")

        # Copy source demolocation into GRASS database
    try:
        copytree(src, dst, ignore=ignore_patterns('*.tmpl','Makefile*'))
```

Otherwise, copytree() gives an error message since the dst directory (grassdatabase) already exists.

Reply:
Corrected.